site/usage/webcomponent.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: usage
+menu: usage
+title: Embedding Vega-Lite with vega-element
+permalink: /usage/webcomponent.html
+---
+
+[Web components]((https://www.webcomponents.org/introduction) are a set of web platform APIs that allow you to create new custom, reusable, encapsulated HTML tags to use in web pages and web apps.
+
+[`vega-element`](https://www.webcomponents.org/element/PolymerVis/vega-element) is a Polymer webcomponent that can be used to embed vega or vega-lite visualization using custom HTML tags.
+
+**Quick start**
+The simplest usage case is to just embed a vega-lite visualization.
+```html
+<!-- import vega-element webcomponent from CDN (not for production use!!) -->
+<link rel="import" href="https://rawgit.com/PolymerVis/vega-element/polymer2/vega-element-cdn.html">
+<!-- import vega-tooltip (required only if using vega-tooltip plugin)-->
+<link rel="import" href="https://rawgit.com/PolymerVis/vega-element/polymer2/vega-tooltip.html">
+
+<!--
+hover flag enables hover event processing
+tooltip flag enables vega-tooltip plugin
+-->
+<vega-element hover tooltip vega-lite-spec-url="vl-spec.json"></vega-element>
+```
+
+Export SVG or PNG
+```html
+<!-- import vega-element webcomponent from CDN (not for production use!!) -->
+<link rel="import" href="https://rawgit.com/PolymerVis/vega-element/polymer2/vega-element-cdn.html">
+
+<!-- do not render anything when headless flag is set -->
+<vega-element id="chart" headless vega-lite-spec-url="vl-spec.json"></vega-element>
+
+<!-- buttons to trigger download -->
+<button onclick="javascript:download('svg')">export SVG</button>
+<button onclick="javascript:download('png')">export PNG</button>
+```
+```javascript
+function download(type) {
+  documentation.querySelector('#chart').downloadImage(type);
+}
+```
+
+vega signals and data can be also be reactively updated using `vega-signal`, `vega-data`, and `vega-data-stream` webcomponents. More examples and full documentation on `vega-element` can be found @
+[`here`](https://www.webcomponents.org/element/PolymerVis/vega-element)
+
+**Blocks examples**
```

Comment:
Block, no plural, I believe.

Reply:
https://bl.ocks.org/-/about

Bl.ocks (pronounced “Blocks”) is a simple viewer for sharing code examples hosted on GitHub Gist. 

I think it depends on the interpretation. Not sure what is the official stand. I am fine with "Block" if that is preferred.